source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,243 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to perform a **text search** in MongoDB.
+
+A text search allows you to search for partial or exact matches from a set
+of words. You can use this for searching through unstructured text, which is
```

Comment:
I'm not sure partial or exact matches describes what it's matching. Instead, how about the following:

```suggestion
You can use a text search to search documents for terms or phrases. A term is a sequence of characters that excludes whitespace characters. A phrase is a sequence of characters that may include any number of whitespace characters. You can use this for searching through unstructured text, which is
```